packages/openneuro-server/graphql/resolvers/dataset.js

```diff
@@ -13,6 +13,7 @@ import { metadata } from './metadata.js'
 import * as dataladAnalytics from '../../datalad/analytics.js'
 import DatasetModel from '../../models/dataset.js'
 import fetch from 'node-fetch'
+import * as Sentry from '@sentry/browser'
```

Comment:
This should be @sentry/node. browser is not installed for the server, so a clean node_modules leads to a server crash.

Reply:
Ah thanks. The dangers of mindless copy-paste ;)